pkg/server/endpoints/registration/handler.go

```diff
@@ -30,10 +31,16 @@ func (h *Handler) CreateEntry(
 	response *registration.RegistrationEntryID, err error) {
 
 	dataStore := h.Catalog.DataStores()[0]
+
+	err = h.checkEntryUniqueness(dataStore, request)
+	if err != nil {
+		h.Log.Error(err)
+		return nil, errors.New("Error trying to create entry")
```

Comment:
nit: errors should be lowercased. Fixing this probably impacts above log line too in order to keep things clean

ref: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Yeah, the thing is that there are a lot of errors in this file written as this one...should I change them all? I can do it, just want confirmation.

Reply:
Oh... boo :) yea don't bother, don't want the diff to get huge... I hope that one day we will majorly refactor this handler